docs/messaging.rst

```diff
@@ -170,7 +174,7 @@ Every message is serialized to a sequence of at least six blobs of bytes:
       b'{parent_header}', # serialized parent header dict
       b'{metadata}',      # serialized metadata dict
       b'{content}',       # serialized content dict
-      b'blob',            # extra raw data buffer(s)
+      b'0x1234',          # extra raw data buffer(s)
```

Comment:
Added #212, making it a non-ascii binary string, to clarify that it's raw bytes.